### DISCUSSION TITLE
Anyone else using Discussions to greatly improve their project management?

### DISCUSSION THREAD

**Discussion by gilgongo** [community/community#3878] [Discussions]
For years on public repos, GitHub Issues was a free-for-all grab bag of bug reports, feature requests, general chat - frantically tagged to the rafters by admins trying to sort out the signal from the noise. Many, many projects suffered: people would post questions and get no answers as they got buried under bug reports or feature requests. The latter often started out as "Let's do X" but then 100 comments later it would have morphed into "Let's do Y" but nobody who wasn't in the conversation knew that.

It was chaos.

Now, with Discussions, we can finally (almost, see below) maintain Issues as actionable work specs prior to the creation of PRs! It's amazing! If somebody posts an idea to Issues that hasn't been discussed or agreed, we move it to Discussions until the outcome of that is ready for an Issue or a PR! Yay!

But there is still a problem: we can't prevent randos posting Issues when they should be Discussions. Please can we have a permissions level for this?

![bitmoji](https://sdk.bitmoji.com/render/panel/8e8dbcd6-ea6f-4dcf-8295-1e999a9d1884-e453f381-f13d-466a-a20e-c03c786b333b-v1.png?transparent=1&palette=1&width=246)

**Comment by MrCrypticxDev** [comment]
Yeah a great feature!
